src/native/managed/cdacreader/Microsoft.Diagnostics.DataContractReader.Contracts/Legacy/ICorProfiler.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Marshalling;
+
+namespace Microsoft.Diagnostics.DataContractReader.Legacy;
+
+internal enum COR_PRF_MONITOR
```

Comment:
This file name seems off.

Reply:
This is meant to follow the same naming convention as the other files in this directory: the file is named after the main COM interface and includes the interface definition plus any structs and enums that woudl normally be in the same .idl file.  Except in this case I didn't need any of the rest of ICorProfiler, so I didn't add it.

I can add it

Reply:
On second thought this is ridiculous. I'll just move the enun into ReJIT_1, same as other implementation details flags that go with a contract